ci/scripts/connect-release.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+# Usage:
+# ./connect-release.sh DESTINATION
+# @DESTINATION: required, destination directory (connect major version)
+
+# Validate params
+if [ -z "$1" ]; then
+    echo "Invalid params"
+    echo "Usage example: ./s3sync.sh 9"
+    exit 1
+fi
+
+echo "Uploading to s3://connect.trezor.io/$1/"
```

Comment:
personally, I would asign some default (=9) to $1 so that we dont screw up accidentaly

Reply:
maybe even forbid anything different than `9` at the moment. I am just super anxious when touching these production-grade things